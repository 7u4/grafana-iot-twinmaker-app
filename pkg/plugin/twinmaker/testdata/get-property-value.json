{
    "PropertyValues": {
        "alarm_key": {
            "PropertyReference": {
                "ComponentName": "AlarmComponent",
                "EntityId": "Mixer_1_4b57cbee-c391-4de6-b882-622c633a697e",
                "ExternalIdProperty": null,
                "PropertyName": "alarm_key"
            },
            "PropertyValue": {
                "BooleanValue": null,
                "DoubleValue": null,
                "Expression": null,
                "IntegerValue": null,
                "ListValue": null,
                "LongValue": null,
                "MapValue": null,
                "RelationshipValue": null,
                "StringValue": "Mixer_1_597c735b-38fd-476c-b276-7592b1699ef8"
            }
        },
        "telemetryAssetType": {
            "PropertyReference": {
                "ComponentName": "AlarmComponent",
                "EntityId": "Mixer_1_4b57cbee-c391-4de6-b882-622c633a697e",
                "ExternalIdProperty": null,
                "PropertyName": "telemetryAssetType"
            },
            "PropertyValue": {
                "BooleanValue": null,
                "DoubleValue": null,
                "Expression": null,
                "IntegerValue": null,
                "ListValue": null,
                "LongValue": null,
                "MapValue": null,
                "RelationshipValue": null,
                "StringValue": "Alarm"
            }
        }
    }
}