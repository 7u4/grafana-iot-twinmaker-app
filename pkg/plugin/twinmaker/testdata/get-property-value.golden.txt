ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] 
Name: 
Dimensions: 2 Fields by 1 Rows
+---------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------+
| Name: alarm_key                                                                             | Name: telemetryAssetType                                                                    |
| Labels: componentName=AlarmComponent, entityId=Mixer_1_4b57cbee-c391-4de6-b882-622c633a697e | Labels: componentName=AlarmComponent, entityId=Mixer_1_4b57cbee-c391-4de6-b882-622c633a697e |
| Type: []*string                                                                             | Type: []*string                                                                             |
+---------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------+
| Mixer_1_597c735b-38fd-476c-b276-7592b1699ef8                                                | Alarm                                                                                       |
+---------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////gAIAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEDAAoADAAAAAgABAAKAAAACAAAAFAAAAACAAAAKAAAAAQAAABk/v//CAAAAAwAAAAAAAAAAAAAAAUAAAByZWZJZAAAAIT+//8IAAAADAAAAAAAAAAAAAAABAAAAG5hbWUAAAAAAgAAAAwBAAAEAAAADv///xQAAADIAAAAyAAAAAAABQHEAAAAAgAAADgAAAAEAAAA1P7//wgAAAAcAAAAEgAAAHRlbGVtZXRyeUFzc2V0VHlwZQAABAAAAG5hbWUAAAAABP///wgAAABoAAAAXAAAAHsiY29tcG9uZW50TmFtZSI6IkFsYXJtQ29tcG9uZW50IiwiZW50aXR5SWQiOiJNaXhlcl8xXzRiNTdjYmVlLWMzOTEtNGRlNi1iODgyLTYyMmM2MzNhNjk3ZSJ9AAAAAAYAAABsYWJlbHMAAAAAAAD8/v//EgAAAHRlbGVtZXRyeUFzc2V0VHlwZQAAAAASABgAFAATABIADAAAAAgABAASAAAAFAAAAMgAAADMAAAAAAAFAcgAAAACAAAAOAAAAAQAAADY////CAAAABQAAAAJAAAAYWxhcm1fa2V5AAAABAAAAG5hbWUAAAAACAAMAAgABAAIAAAACAAAAGgAAABcAAAAeyJjb21wb25lbnROYW1lIjoiQWxhcm1Db21wb25lbnQiLCJlbnRpdHlJZCI6Ik1peGVyXzFfNGI1N2NiZWUtYzM5MS00ZGU2LWI4ODItNjIyYzYzM2E2OTdlIn0AAAAABgAAAGxhYmVscwAAAAAAAAQABAAEAAAACQAAAGFsYXJtX2tleQAAAAAAAAD/////2AAAABQAAAAAAAAADAAWABQAEwAMAAQADAAAAEgAAAAAAAAAFAAAAAAAAAMDAAoAGAAMAAgABAAKAAAAFAAAAHgAAAABAAAAAAAAAAAAAAAGAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACAAAAAAAAAAIAAAAAAAAADAAAAAAAAAAOAAAAAAAAAAAAAAAAAAAADgAAAAAAAAACAAAAAAAAABAAAAAAAAAAAgAAAAAAAAAAAAAAAIAAAABAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAAAAAAsAAAATWl4ZXJfMV81OTdjNzM1Yi0zOGZkLTQ3NmMtYjI3Ni03NTkyYjE2OTllZjgAAAAAAAAAAAUAAABBbGFybQAAABAAAAAMABQAEgAMAAgABAAMAAAAEAAAACwAAAA4AAAAAAADAAEAAACQAgAAAAAAAOAAAAAAAAAASAAAAAAAAAAAAAAAAAAAAAAACgAMAAAACAAEAAoAAAAIAAAAUAAAAAIAAAAoAAAABAAAAGT+//8IAAAADAAAAAAAAAAAAAAABQAAAHJlZklkAAAAhP7//wgAAAAMAAAAAAAAAAAAAAAEAAAAbmFtZQAAAAACAAAADAEAAAQAAAAO////FAAAAMgAAADIAAAAAAAFAcQAAAACAAAAOAAAAAQAAADU/v//CAAAABwAAAASAAAAdGVsZW1ldHJ5QXNzZXRUeXBlAAAEAAAAbmFtZQAAAAAE////CAAAAGgAAABcAAAAeyJjb21wb25lbnROYW1lIjoiQWxhcm1Db21wb25lbnQiLCJlbnRpdHlJZCI6Ik1peGVyXzFfNGI1N2NiZWUtYzM5MS00ZGU2LWI4ODItNjIyYzYzM2E2OTdlIn0AAAAABgAAAGxhYmVscwAAAAAAAPz+//8SAAAAdGVsZW1ldHJ5QXNzZXRUeXBlAAAAABIAGAAUABMAEgAMAAAACAAEABIAAAAUAAAAyAAAAMwAAAAAAAUByAAAAAIAAAA4AAAABAAAANj///8IAAAAFAAAAAkAAABhbGFybV9rZXkAAAAEAAAAbmFtZQAAAAAIAAwACAAEAAgAAAAIAAAAaAAAAFwAAAB7ImNvbXBvbmVudE5hbWUiOiJBbGFybUNvbXBvbmVudCIsImVudGl0eUlkIjoiTWl4ZXJfMV80YjU3Y2JlZS1jMzkxLTRkZTYtYjg4Mi02MjJjNjMzYTY5N2UifQAAAAAGAAAAbGFiZWxzAAAAAAAABAAEAAQAAAAJAAAAYWxhcm1fa2V5AAAAqAIAAEFSUk9XMQ==
